libs/core/scripts/build-tokens.ts

```diff
@@ -0,0 +1,84 @@
+import fs from 'fs-extra';
+import * as tokens from '../tokens/tokens.json';
+
+/**
+ * 
+ * @param tokens Tokens data in JSON format.
+ * @param base Base string to format custom properties.
+ * @returns Formatted CSS with all custom properties.
+ */
+const tokensToCss = (tokens: object, base = `-`) =>
+  Object.entries(tokens).reduce((css, [key, value]) => {
+		let newBase = base + `-${key}`;
```

Comment:
One more thing I just thought of. Thoughts on converting the camelcase JSON names to dashes/kebab-case?

So rather than `--default-fontWeights-headingBold` it'd be `--default-font-weights-heading-bold`. Since custom props are case-sensitive it seems less likely to cause issues.

Reply:
Appreciate the callout here. We can revisit this as a group once all the tokens are in place in Figma.